scripts/extract_documents.py

```diff
@@ -0,0 +1,66 @@
+""" A script to extract raw and rendered versions of all documents in the db
```

Comment:
This is the script I used to render all the documents in my local db, to verify that the new versino of KumaScript produces the same output as the old version.